api-models/thrift/src/main/thrift/appsRendering.thrift

```diff
@@ -3,6 +3,19 @@ include "content/v1.thrift"
 namespace scala com.gu.mobile.ar.models
 namespace typescript _at_guardian.apps_rendering_api_models
 
+struct Image {
+    1: required string url
```

Comment:
What type of url is provided for us here? Will it be a master asset that we can derive a `srcset` from?

Reply:
Found it, the answer is "yes", it is a master asset.